chore: adds 4.16.0 release notes (#1739)
medic/cht-docs: content/en/core/releases/4.16.0.md

---
title: "4.16.0 release notes"
linkTitle: "4.16.0"
weight:
description: >
relevantLinks: >
toc_hide: true
---

## Known issues

Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+4.16.0%22).

## Upgrade notes

### Breaking changes

None.

### UI/UX changes

None.

## Highlights

### Upgrading to CouchDb 3.4.2

> CouchDb 3.4.2 comes bundled by default with CHT-Core 4.16.0 and includes a multitude of improvements, including:

Reviewer: ```suggestion
CouchDb 3.4.2 comes bundled by default with CHT Core 4.16.0 and includes a multitude of improvements, such as:
```

_nitpick_: the `include` word is used twice in the sentence; replacing it with something else.

- Adds Nouveau, an implementation of Lucene, which will be used in the near future for improved freetext searching
- Improved security by adding a new, more secure and more performant password hashing algorithm 

> - An alternative Javascript view engine, which has improved performance when indexing views. The CHT is not yet ready for switching to this engine, as the two engines are not fully compatible, but will be in a future release.  

Reviewer: ```suggestion
- An alternative JavaScript view engine, which has improved performance when indexing views. The CHT is not yet ready for switching to this engine, as the two engines are not fully compatible, but will be in a future release.  
```

_nitpick_

> The upgrade to the new version of CouchDb should be seamless and covered by the generic CHT-Core upgrade, without additional intervention. 

Reviewer: ```suggestion
The upgrade to the new version of CouchDb should be seamless and covered by the generic CHT Core upgrade, without additional intervention. 
```
_nitpick_

### Arabic as CHT language

> The CHT is soon expanding its reach through adding support for Right-To-Left written languages. As part of the initial effort on this path, version 4.16.0 adds Arabic translations for Webapp, while the required UI changes for complete RTL support to be added in a future version.  

Reviewer: ```suggestion
The CHT is soon expanding its reach through adding support for Right-To-Left written languages. As part of the initial effort on this path, version 4.16.0 adds Arabic translations for Webapp, while the required UI changes for complete RTL support are to be added in a future version.  
```
_nitpick_
